kuma/static/js/contribution-handler.js

```diff
@@ -359,9 +360,10 @@
 
         // Send GA Event.
         mdn.analytics.trackEvent({
-            category: 'Contribution submission',
-            action: isPopoverBanner ? 'On Popover' : 'On Page',
-            value: 1
+            category: 'payments',
+            action: 'submission',
```

Comment:
Note to future jwhitlock: This event is fired when the Stripe popup is shown, not when the callback is called. A user can cancel, not complete payment.

Reply:
Maybe I can create an event for submission (in the token callback) and an event for closing the modal (in the close callback)?

Reply:
I think that is a good idea. That way we can also see how many users initiate the stripe dialog but never complete payment.

Reply:
I did have a simple solution where I just throw an event in the checkout.js callbacks for `close` and `token` I just pushed, but I'm happy to remove it if there's too much complexity, but looking at the above solution it would be good to include that after this release?

Reply:
👍 The "completed" event worked for me